fbmessenger/__init__.py

```diff
@@ -174,8 +179,8 @@ def send_generic_template(self, payload, recipient_id, timeout=None):
                                         },
                                         {
                                             "type": "postback",
-                                            "title": "Start Chatting",
-                                            "payload": "DEVELOPER_DEFINED_PAYLOAD",
+                                            "title": "Like",
+                                            "payload": "https://www.youtube.com/watch?v=S_1WBzXFyBs&t=3752s",
```

Comment:
@linbeta review this PR is highly recommended (if you want to know what's going on under the postback hood :) )

the main logic is as follow:
1. make sure our `generic template` has a `postback` payload with the data we want, in this case it's `like` with the content url that user clicked!

Reply:
I am not quite sure how this template works yet. I would pull these new updates and run some tests to figure it out. Thanks for helping out!